File: Swift-C++-IR-Comparison/test/controlFlow/Swift/SwitchNumbers.swift
@main
struct MyMain {
    
    static func main() {
        let res = switchMe(input: 1)
    }
    
    static func switchMe(input: Int) -> Int {
        var counter = 0
        var max = 7
        switch input {
        case 1:
            counter = max
        case 0:
            counter = -max
        default:
            counter = 42
        }
        return counter
    }
}

File: Swift-C++-Testsuite/test/controlFlow/C++/SwitchNumbers.cpp

   
   static int switchMe(int input) {
       int counter = 0;
       int max = 7;
       
       switch(input) {
           case 1:
               counter = max;
               break;
           case 0:
               counter = -max;
               break;
           default:
               counter = 42;
       }
       return counter;
   }


 int main() {
       int res = switchMe(1);
       return 0;
   }
